### DISCUSSION TITLE
Can sandboxie have different mac for each sandbox

### DISCUSSION THREAD

**Discussion by vipnewbai** [sandboxie-plus/Sandboxie#3257] [General]
I want different sandbox have different mac ，

Which boss can help me achieve

**Comment by vipnewbai** [comment]
i can pay money

**Comment by bastik-1001** [comment]
It is possible to set a `NetworkAddress` at `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Class\{4D36E972-E325-11CE-BFC1-08002BE10318}\[device_ID]` inside each sandbox. I can't confirm that this has any effect, so it is unclear if that works. Maybe someone else has deeper insight into this.

**Reply by e-t-l** [reply]
@vipnewbai I would be interested to know if this works for you.

**Comment by DavidXanatos** [comment]
How easy that is propably depands in how the client software is checkign the mac, there are a few API's that allow for that.
